### ISSUE TITLE
Bug for some in test_converters.py

### ISSUE THREAD

**Issue by dcolligan** [ga4gh/server#727]
```
nosetests -vs tests/unit/test_converters.py
testSamConversion (tests.unit.test_converters.TestSamConverter) ... ERROR

======================================================================
ERROR: testSamConversion (tests.unit.test_converters.TestSamConverter)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dannyc/ga4gh/server/tests/unit/test_converters.py", line 69, in testSamConversion
    self.verifyFullConversion(readGroupSet, readGroup, reference)
  File "/Users/dannyc/ga4gh/server/tests/unit/test_converters.py", line 43, in verifyFullConversion
    converter.convert()
  File "/Users/dannyc/ga4gh/server/ga4gh/converters.py", line 71, in convert
    alignedSegment = SamLine.toAlignedSegment(read, targetIds)
  File "/Users/dannyc/ga4gh/server/ga4gh/converters.py", line 129, in toAlignedSegment
    ret.flag = cls.toSamFlag(read)
  File "/Users/dannyc/ga4gh/server/ga4gh/converters.py", line 163, in toSamFlag
    if read.nextMatePosition.strand == protocol.Strand.NEG_STRAND:
AttributeError: 'NoneType' object has no attribute 'strand'
```

**Comment by dcolligan** [comment]
read `dataset1:chr17.1-250:fish:x3` has a null `nextMatePosition`, which is allowed by the schema

**Comment by dcolligan** [comment]
Fixed in #728
